lib/resources.js

```diff
@@ -343,6 +344,18 @@ module.exports = function (app, _private = null) {
     return body
   }
 
+  const addItemQuery = (body, itemUri) => {
+    body.query.bool.must.push({
+      nested: {
+        path: 'items',
+        query: { bool: { must: [{ term: { 'items.uri': itemUri } }] } },
```

Comment:
Couldn't this just be `        query: { term: { 'items.uri': itemUri } },` ?

Reply:
Fixed